Requirements.md

```diff
@@ -0,0 +1,13 @@
+## Software Requirements Specifications for Spring 2020 "Intro to Software Engineering" "Craft Topography" Project at Wright State Univesity
+### Functional Requirements
+  #### User Story A
```

Comment:
Following Waterfall process, Task flow from requirements. There will typically be one or more tasks necessary to complete each Requirement.